python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -175,7 +175,7 @@ def fixSubsetValidity(self, conn, table, detectorData, validity):
                 if valids[date][1] > midpoint:
                     nextDate = dates[i + 1]
                     valids[nextDate][0] = midpoint + datetime.timedelta(1)
-                    valids[date][1] = midpoint
+                    valids[date][1] = midpoint + datetime.timedelta(1)
```

Comment:
Did you find a way to confirm that this part of the fix is correct experimentally?

Reply:
My initial change removed the `+ datetime.timedelta(1)` on line 177.  However, this resulted in both `validEnd_i` and `validStart_{i+1}` being one day early.  That was clearly wrong, as the `calibDate` was not contained within the range, leading to this version that increments both fields.